spec/models/ride_spec.rb

```diff
@@ -2,12 +2,21 @@
 
 describe Ride do
   let!(:ride) { create :ride }
-  let!(:user) { create :user }
+  let(:driver) { create :user }
+  let!(:passenger) { create :user }
+
+  before(:each) do
+    ride.add_driver driver
+  end
+
+  it 'has a driver' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.